app/views/movie/trailer.jsx

```diff
@@ -0,0 +1,21 @@
+//libraries
+import React, {PropTypes} from 'react';
+
+export default React.createClass({
+    displayName: 'Trailer',
+    propTypes: {
+        url: PropTypes.string
+    },
+
+    /** @inheritDoc */
+    render() {
+        const {url} = this.props;
+        return (
+            <div data-demo='trailer'>
+                {url &&
+                    <iframe src={url} allowfullscreen={true} seamless={true} width="1024" height="768" />
```

Comment:
No, in HTML 5 spec of iframe, it is mentionned that you can only specify pixels.